app/models/facility.rb

```diff
@@ -255,6 +255,10 @@ def _destroy
     end
   }
 
+  def discardable?
```

Comment:
Let's slap some test coverage on these `discardable` methods.

Reply:
Would there be cases were we want to cascade the discard to the associated records here?  I can imagine that for 90%+ of cases, there will be some associated records on facility or facility groups, so this would almost always block the discard.

Reply:
We don't want to cascade discards, we only want to allow a discard from the UI is brand new organizations/facility groups/facilities that were created accidentally etc.

The other case is when there are duplicate facilities, but those need the data migrated manually before we allow the discard.

Reply:
I suspect this will allow for deletion if you _just_ set something up and you wanna trash it.

Reply:
Deleting a facility/group that's in use is... much bigger :/ I don't think it'll be a usable feature since we don't have a way to, say, move patients from one facility to another...